drip/models.py

```diff
@@ -247,8 +297,17 @@ def filter_kwargs(self, qs, now=datetime.now):
 
         return kwargs
 
-    def apply(self, qs, now=datetime.now):
-
+    def apply(self, qs: AbstractQuerySetRuleQuerySet, now: DateTime = datetime.now) -> AbstractQuerySetRuleQuerySet:
+        """
+        Returns qs filtered/excluded by any filter resulting from self.filter_kwargs
+        depending on whether self.method_type is one of the following:
+            * 'filter'
+            * 'exclude'
+        Also annotates qs by calling self.apply_any_annotation.
+        :param qs:
+        :param now:
+        :return:
```

Comment:
do we need these now that the functions are type annotated?

Reply:
my bad, thanks for noticing it.